### PR TITLE
Spanish translations for the accessibility strings

### LINKED ISSUE #468: localize various accessibility text
In #444 , the translations for the home page were moved to our shared package. We're now ready to enable localization to the rest of the web package.

## Translations

https://docs.google.com/document/d/12JxxvJcIySxBMensv48UrhiG0fkaLcCLiUttuuhv8Uk/edit?usp=sharing

## Content

- "[Current Temperature](https://github.com/CodeForBoulder/upswyng/blob/cf0e6bea6bdd2c02278c30ea9f952fd1143c3a5e/packages/web/src/components/Temperature.tsx#L44)"
- "[go back to previous page](https://github.com/CodeForBoulder/upswyng/blob/cf0e6bea6bdd2c02278c30ea9f952fd1143c3a5e/packages/web/src/components/BackButton.tsx#L21)"
- "[menu](https://github.com/CodeForBoulder/upswyng/blob/cf0e6bea6bdd2c02278c30ea9f952fd1143c3a5e/packages/web/src/components/Header.tsx#L26)"
- "[close alerts](https://github.com/CodeForBoulder/upswyng/blob/cf0e6bea6bdd2c02278c30ea9f952fd1143c3a5e/packages/web/src/components/AlertsDrawer.tsx#L53)"

## Checklist
- [x] translations for both English and Spanish were added to the [shared package](https://github.com/CodeForBoulder/upswyng/tree/master/packages/common/src/i18n)
- [x] translation keys were substituted for the correct content

### PR DESCRIPTION
**Pull request by cjlynch278**
This PR closes #468

## What does this PR do?
Add translations to the shared packages and substitute the correlating strings

### CODE REVIEW

**Comment by jollyjerr** [comment]
Hey @cjlynch278 I see you are still committing regularly to this branch so let me know when you would like a review!

**Comment by cjlynch278** [comment]
Hey @jollyjerr should be all set! I just wanted to make sure it passed the updated CI tests that merged

**Review by jollyjerr — APPROVED** [review]
This looks great to me! Nice work @cjlynch278 ! You can merge this whenever! 🎉 🎈 🚀 :shipit:

**Comment by cjlynch278** [comment]
@jollyjerr I don't have writing access super powers to be able to merge.

File: packages/web/src/components/Temperature.tsx [hunk 1]
```diff
@@ -41,7 +43,7 @@ const Temperature = () => {
   return (
     <div className={classes.container}>
       <Typography variant="srOnly" component="h2">
-        Current Temperature
+        {t("backToPrevious")}
```
Comment (jollyjerr) [inline packages/web/src/components/Temperature.tsx:46]:
```suggestion
        {t("currentTemperature")}
```

**Review by jollyjerr — COMMENTED** [review]
Hey sorry for the back and forth! I was doing one last look over this morning and noticed that the temperature component is pointing to the wrong text 😅 . If you can just make these two little changes then we will be good to go

**Comment by cjlynch278** [comment]
@jollyjerr  Ah shoot, sloppy mistake. I just comitted your suggestions

**Pull request merged**